getting_started.rst

```diff
@@ -81,6 +81,60 @@ An MD5 Encrypter using the Poco Libraries
         short_paths: False
         apply_env: True
         build_policy: None
+        settings: ('os', 'arch', 'compiler', 'build_type')
+        options
+            cxx_14: [True, False]
+            enable_apacheconnector: [True, False]
+            enable_cppparser: [True, False]
```

Comment:
I think this output is too much for the getting started section.
Please list the fist two options and default options and write `...` below them:

```
    options:
        shared: [True, False]
        enable_xml: [True, False]
        ...
    default_options: 
        shared=False
        enable_xml=True
        ...
```

Reply:
Why are options and default options in different order?

Reply:
I extracted from command output, probably is related to declared order.

Reply:
Done!